app.py

```diff
@@ -0,0 +1,26 @@
+from flask import Flask, render_template, request
+import sys
+
+sys.path.insert(0, 'src')
+
+from contacts_book import *
+from contact import * 
```

Comment:
You should be able to use `import contacts_book` instead of `from contacts_book import *` I think? (And same applies to other imports)